website_crm_recaptcha/controllers/main.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.http import route
+from openerp.exceptions import ValidationError
+from openerp.addons.website_form_recaptcha.controllers.main import WebsiteForm
+from openerp.addons.website_crm.controllers.main import contactus
+
+
+class ContactUsRecaptcha(WebsiteForm, contactus):
+    @route()
+    def contactus(self, **kwargs):
```

Comment:
In v9, [`website_crm` uses the controller from `website_form` to process the requests](https://github.com/odoo/odoo/blob/9.0/addons/website_crm/__openerp__.py#L12). V8 has its own, and does not have `website_form`, so this server-side validation must be added here.